firefox-ios/EcosiaTests/Account/Mocks/MockNativeToWebSSOAuth0Provider.swift

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Auth0
+@testable import Ecosia
+
+/// A mock that simulates NativeToWebSSOAuth0Provider behavior for testing
+/// Note: Since NativeToWebSSOAuth0Provider is a struct, we can't inherit from it
+/// Instead, we create a separate mock that can be used in tests that need SSO functionality
+final class MockNativeToWebSSOAuth0Provider: MockAuth0Provider {
```

Comment:
Not sure I get why this is needed, also couldn't find it being used, can you elaborate?

Reply:
Removing as per https://github.com/ecosia/ios-browser/pull/925#discussion_r2204448827